tests/core/api/GH_ORG_SUBS_API.js

```diff
@@ -0,0 +1,436 @@
+                new Error(
+                  util.format('User cannot get subscription',
+                    query, err)
+                )
+              );
+            subscriptions = subs;
+            assert.isNotEmpty(subscriptions, 'User cannot find the subscriptions');
+            assert.equal(subscriptions.length,
+              global.ADM_GH_SUB_COUNT, 'Subscription count needs to match');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('2. Owner can get subscription by ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        ownerApiAdapter.getSubscriptionById(sub.id,
+          function (err, subAcct) {
+            if (err || _.isEmpty(subAcct))
+              return done(
+                new Error(
+                  util.format('User cannot get subscription Account from ID',
+                    query, err)
+                )
+              );
+            subscription = subAcct;
+            assert.isNotEmpty(subscription, 'User cannot find the subscription by ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('3. Member can get subscription by ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        memberApiAdapter.getSubscriptionById(sub.id,
+          function (err, subAcct) {
+            if (err || _.isEmpty(subAcct))
+              return done(
+                new Error(
+                  util.format('User cannot get subscription Account from ID',
+                    query, err)
+                )
+              );
+            subscription = subAcct;
+            assert.isNotEmpty(subscription, 'User cannot find the subscription by ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('4. Collaborator can get subscription by ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        ownerApiAdapter.getSubscriptionById(sub.id,
+          function (err, subAcct) {
+            if (err || _.isEmpty(subAcct))
+              return done(
+                new Error(
+                  util.format('User cannot get subscription Account from ID',
+                    query, err)
+                )
+              );
+            subscription = subAcct;
+            assert.isNotEmpty(subscription, 'User cannot find the subscription by ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('5. Public user cannot get subscription by ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
```

Comment:
we are missing one more test here. We should that public users cannot get any values if the make a getS call without Ids. We should also test getS call by passing a subscriptionId